.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
i didn't get the reason of this `idea` folder!

Reply:
something used inside the webstorm ide which u don't know cuz ur using vscode even if he has .vscode file